bindings/c/test/mako/mako.c

```diff
@@ -1082,7 +1101,12 @@ int run_workload(FDBTransaction* transaction,
 		free(keystr);
 		return -1;
 	}
-	valstr = (char*)malloc(sizeof(char) * args->value_length + 1);
+	int value_length = args->value_length;
```

Comment:
Oh, I thought we're changing the transaction size, not the value.
Changing the value size might work for our purpose though.

Reply:
Oh I thought we should increase the value size. I guess I'm mistaken. Do you think we should modify the operation execution number rather than the value size?

Reply:
`value_length` is not used when generating the value

Reply:
Thanks for this notification. It seems the `run_one_transaction` function still rely on the args->value_length to generating the value